arangod/Cluster/MaintenanceFeature.h

```diff
@@ -308,29 +308,6 @@ class MaintenanceFeature : public application_features::ApplicationFeature {
    */
   void delShardVersion(std::string const& shardId);
 
-  /**
```

Comment:
We should also delete the instance variables `_currentCounterLock`, `_currentCounterCondition` and `_currentCounter`.
They don't seem to be necessary after this change.